tests/end_to_end/andor_mixed/orig.c

```diff
@@ -8,7 +8,23 @@ int test(int a, int b, int c, int d) {
 
```

Comment:
can you add a test for `if (a && (b ? c : d) == e)` to check that nothing bad happens?